### PR TITLE
austin.intermediate.summer2.kennedy.taylor

### PR DESCRIPTION
**Pull request by TaylorKennedy**

### CODE REVIEW

File: apps/01PigLatin.js [hunk 1]
```diff
@@ -7,17 +7,97 @@ prompt.start();
...
+  //the vowel free firstPart and the restWord, which starts with the first vowel
+  var firstPart = word.slice(0, vowelIndex);
+  var restWord = word.slice(vowelIndex, word.length);
+
+  if (vowelIndex === 0) {
+    return word + 'yay';
+  }
+  else {
+    return restWord + firstPart + 'ay';
+  }
```
Comment (pieroapretto) [inline apps/01PigLatin.js:74]: App looks good!

File: homework/01LessonOne.js [hunk 2]
```diff
@@ -106,7 +107,8 @@ var notNotTrue;
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
 // Your Answer Goes Here:
-//
+//"and" checks for both arguments to be true, "or" checks for at least one argument being true.
+//"and" should be used when two things need to be true together, "or" should be used when one or more thing needs to be true but it doesn't matter which one
 // ****
```
Comment (pieroapretto) [inline homework/01LessonOne.js:112]: Good answer!

File: homework/01LessonOne.js [hunk 3]
```diff
@@ -203,12 +206,12 @@ var stringFourStrictNotEqualsNumberFour;
...
 // Your Answer Goes Here:
-//
+//== checks that two values are equivalent. === is a stricter check that needs the two arguments to have equal value and the same type
 //
 // What’s the difference between != and !==?
 //
 // Your Answer Goes Here:
-//
+////!= evaluates to true if the two arguments differ in value, !== evaluates to true if the two arguments differ in value or in type
 // ****
```
Comment (pieroapretto) [inline homework/01LessonOne.js:215]: Very good!

File: apps/01PigLatin.js [hunk 4]
```diff
@@ -7,17 +7,97 @@ prompt.start();
...
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:
+  //toLowerCase() will convert the string to all lower case letters, so that we don't have to worry about capitalizataion when we translate the word
```
Comment (paulrenenichols) [inline apps/01PigLatin.js:17]: Awesome!

fist bump

File: apps/01PigLatin.js [hunk 5]
```diff
@@ -7,17 +7,97 @@ prompt.start();
...
 
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer:
+  //vowelIndex will be used for the position in the word where the first vowel occurs.
+  //Since -1 will never be a valid position, we can use it to represent "no vowel found"
```
Comment (paulrenenichols) [inline apps/01PigLatin.js:28]: Good.

File: apps/01PigLatin.js [hunk 6]
```diff
@@ -7,17 +7,97 @@ prompt.start();
...
+  //
+  // Why are we using String.prototype.slice() here? What role does vowelIndex play?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice
+  //
+  // Your Answer:
+  //slice(startIndex, endIndex) returns a substring of the word that starts with
+  //the character in position startIndex and ends just before the character in position endIndex.
+  //vowelIndex is the position where the first vowel is, so we are slicing the word up into
+  //the vowel free firstPart and the restWord, which starts with the first vowel
```
Comment (paulrenenichols) [inline apps/01PigLatin.js:65]: Looks like someone is reading the documentation on mdn. yay!

File: apps/01PigLatin.js [hunk 7]
```diff
@@ -7,17 +7,97 @@ prompt.start();
...
+// http://intermediate.austincodingacademy.com/_book/lesson_one.html
+//
+// What does prompt.get() do for us?
+//
+// Your Answer:
+//prompt.get() displays a message to the user ('word') and then grabs the text that the user inputs and stores it (in result).
 function getPrompt() {
     prompt.get(['word'], function (error, result) {
-
+      if (result['word'].toLowerCase() != "q"){
```
Comment (paulrenenichols) [inline apps/01PigLatin.js:90]: Improvements? Haha.

**Pull request closed without merging**